.github/workflows/boxd-golden-refresh.yml

```
@@ -0,0 +1,153 @@
+        run: |
+          set -euo pipefail
+          curl -fsSL --max-time 60 https://boxd.sh/downloads/cli/install.sh -o /tmp/boxd-install.sh
+          echo "$BOXD_INSTALLER_SHA256  /tmp/boxd-install.sh" | sha256sum -c -
+          sh /tmp/boxd-install.sh
+          echo "$HOME/.local/bin" >> "$GITHUB_PATH"
+
+      # Per-run paths: a run cancelled by the job timeout can leave its detached
+      # refresh alive on the golden, and shared file names would let the next
+      # run read the dead run's status.
+      - name: Per-run remote paths
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2129:style:2:1: Consider using { cmd1; cmd2; } >> file instead of individual redirects

(shellcheck)